nawab.py

```diff
@@ -4,32 +4,37 @@
 import pwd
 import os
 import threading
+import argparse
+import logging
 
 #TODO: Shouldn't import this here. Need a way to derive this from tg_bot instead.
 from telegram.ext import Updater, CommandHandler, CallbackQueryHandler, ConversationHandler
 
 
 class Nawab(object):
 
-    def __init__(self, dirpath, data):
+    def __init__(self, dirpath, data, level):
         self.dirpath = dirpath
         self.data = data
+        self.level = level
 
     def retrieve_twitter_auth(self):
-        tw_bot = twitter_bot.Twitter_Bot(self.dirpath, self.data)
+        tw_bot = twitter_bot.Twitter_Bot(self.dirpath, self.data, self.level)
         api = tw_bot.nawab_twitter_authenticate()
         return tw_bot, api
 
 
-    def twitter_bot_run(self):
+    def twitter_bot_run(self, auto_retweet=None):
         tw_bot, api = self.retrieve_twitter_auth()
         tw_bot.nawab_curate_list(api)
-        tw_bot.nawab_retweet_tweet(api)
+        
+        if auto_retweet == True:
+            tw_bot.nawab_retweet_tweet(api)
 
 
-    def tg_bot_run(self):
+    def tg_bot_run(self, auto_retweet=None):
         tw_bot, api = self.retrieve_twitter_auth()
-        bot = tg_bot.Telegram_Bot(api, self.dirpath)
+        bot = tg_bot.Telegram_Bot(api, self.dirpath, self.level, auto_retweet)
```

Comment:
Shouldn't there be a case where auto_retweet is not passed? What would be that condition?

Reply:
No, the value will be none if not passed